jquery.panelSnap.js

```diff
@@ -266,31 +266,35 @@ if ( typeof Object.create !== 'function' ) {
 
       var self = this;
 
-      var nav = self.options.navigation;
-
-      if(!self.enabled) {
+      if(!self.enabled || self.isSnapping) {
         return;
       }
 
-      switch(e.which) {
-        case nav.keys.prevKey:
-        case nav.keys.nextKey:
-          e.preventDefault();
-      }
+      var nav = self.options.navigation;
+      var prev = self.keyIs('prev', e.which);
+      var next = self.keyIs('next', e.which);
 
-      if (self.isSnapping) {
```

Comment:
This check was deliberately this late because the default should still be prevented while panelSnap is doing it's scrolling magic.